test/libsolidity/gasTests/dispatch_large.sol

```diff
@@ -24,9 +24,9 @@ contract Large {
 }
 // ----
 // creation:
-//   codeDepositCost: 904400
-//   executionCost: 942
-//   totalCost: 905342
+//   codeDepositCost: 640200
```

Comment:
This is fully synthetic and unrealistic, but still impressive to see.

Reply:
I wonder why this change does not affect more tests. Only a handful and most of them are gas tests. I'd expect a lot of other tests to be impacted. Are most of other tests below the 100k gas threshold?